src/index.coffee

```diff
@@ -11,7 +11,7 @@ module.exports = class LiveScriptCompiler
 
   compile: (data, path, callback) ->
     try
-      result = LiveScript.compile data, bare: yes
+      result = livescript.compile data, bare: yes, header: no
```

Comment:
that seems to be the scope of another PR, though. This message didn't exist "back then"
(and having the header prevents the test from passing)